src/cms/index.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import Visual from '../components/Visual';
+
+export const TYPES = {
+  HURUMAP_CARD: 'hurumap-card',
+  HURUMAP_CHART: 'indicator-hurumap',
+  FLOURISH_CHART: 'indicator-flourish'
+};
+
+export function dataProps(
+  type,
+  {
+    chartId,
+    chartWidth,
+    title,
+    description,
+    showInsight,
+    insightTitle,
+    insightSummary,
+    dataLinkTitle,
+    analysisCountry,
+    analysisLinkTitle,
+    dataGeoId,
+    geoId,
+    showStatVisual,
+    postId,
+    postType
+  }
+) {
+  return {
+    id: `${type}-${chartId}`,
+    style: {
```

Comment:
1. Changing this causes `Block validation: Block validation failed for ...` for all the charts in Dashboard. WP requires them to be _exactly_ the same. May be [deprecating blocks](https://developer.wordpress.org/block-editor/developers/block-api/block-deprecation/) may help?
2. I'd be happy if we stop using `style` all together and switch to `class`

Reply:
@kilemensi  i created a story for this https://www.pivotaltracker.com/story/show/170817589